internal/driver/device.go

```diff
@@ -35,7 +36,7 @@ var (
 type Device struct {
 	lc                          logger.LoggingClient
 	name                        string
-	path                        string
+	paths                       []interface{}
```

Comment:
we might want to make these be a string array for simplicity, as i believe this is an internal representation

Reply:
So that's what I tried originally, but it kept panicking with type errors. This was the solution I settled on, but I will keep on trying to fix those errors